src/EventSubscriber/StrawberryRunnersEventInsertPostProcessingSubscriber.php

```diff
@@ -0,0 +1,331 @@
+<?php
+
+namespace Drupal\strawberry_runners\EventSubscriber;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Session\AccountInterface;
+use Drupal\strawberryfield\Event\StrawberryfieldCrudEvent;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\StringTranslation\TranslationInterface;
+use Drupal\Core\Messenger\MessengerInterface;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Component\Utility\Unicode;
+use Drupal\file\FileInterface;
+use Drupal\Core\StreamWrapper\StreamWrapperManagerInterface;
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\strawberry_runners\Plugin\StrawberryRunnersPostProcessorPluginManager;
+use Drupal\strawberryfield\EventSubscriber\StrawberryfieldEventInsertSubscriber;
+
+/**
+ * Event subscriber for SBF bearing entity json process event.
+ */
+class StrawberryRunnersEventInsertPostProcessingSubscriber extends StrawberryfieldEventInsertSubscriber {
+
+
+  use StringTranslationTrait;
+
+  /**
+   *
+   * Run as late as possible.
+   *
+   * @var int
+   */
+  protected static $priority = -2000;
+
+  /**
+   * The messenger.
+   *
+   * @var \Drupal\Core\Messenger\MessengerInterface
+   */
+  protected $messenger;
+
+  /**
+   * The entity storage class.
+   *
+   * @var \Drupal\Core\Entity\EntityStorageInterface
+   */
+  protected $storage;
+
+
+  /**
+   * The entity storage class.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface;
+   */
+  protected $entityTypeManager;
+
+  /**
+   * The Config Factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactoryInterface;
+   */
+  protected $configFactory;
+
+  /**
+   * The Stream Wrapper Manager.
+   *
+   * @var \Drupal\Core\StreamWrapper\StreamWrapperManagerInterface
+   */
+  protected $streamWrapperManager;
+
+  /**
+   * An array containing local copies of stored files.
+   *
+   * @var array
+   */
+  protected $instanceCopiesOfFiles = [];
+
+  /**
+   * File system service.
+   *
+   * @var \Drupal\Core\File\FileSystemInterface
+   */
+  protected $fileSystem;
+
+  /**
+   * The logger service.
+   *
+   * @var \Drupal\Core\Logger\LoggerChannelFactoryInterface
+   */
+  protected $loggerFactory;
+
+
+  /**
+   * The StrawberryRunner Processor Plugin Manager.
+   *
+   * @var \Drupal\strawberry_runners\Plugin\StrawberryRunnersPostProcessorPluginManager
+   */
+  protected $strawberryRunnerProcessorPluginManager;
+
+  /**
+   * The current user.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $account;
+
+  /**
+   * StrawberryRunnersEventPreSavePostProcessingSubscriber constructor.
```

Comment:
@DiegoPino Has to be 
StrawberryRunnersEventInsertPostProcessingSubscriber instead of StrawberryRunnersEventPreSavePostProcessingSubscriber? just a silly note

Reply:
Totally. This is silly duplicated code because "save" runs only on existing objects and "insert" only on new ones. So I Copy and Pasta (with pesto) the code. Will fix. Thanks!!